docs/api/course.rst

```diff
@@ -139,7 +139,7 @@ DELETE /api/instructor/<course_id>/<instructor_id>
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 *Remove a course instructor (admins)*
 
-Submissions of the instructor are not removed.
+Submissions of the instructor are not removed from the course's assignments.
```

Comment:
Can we change this to "The instructor's submissions are not removed from the course" or something along those lines.